### PR TITLE
Issue 3346377: Document what features are included

### PR DESCRIPTION
**Pull request by heatherwoz**
## This PR does the following:
- Cleans up heading structure in the main README
- Adds a features.md that summarizes what's included when you install the profile

## Ticket(s)
[Issue 3346377](https://www.drupal.org/project/sous/issues/3346377)

### Functional Testing:
- [ ] Read the docs and see if they make sense.

### Notes:
n/a

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,10 @@
 
 This will provide you with a starting Drupal project that is managed with Composer. The install will include a small set of contrib modules, a starting custom module for specific for the build, and a custom starting theme generated from Emulsify.
```
Comment (birchy82) [inline README.md:8]: "a starting custom module for specific for the build, and a custom starting theme generated from Emulsify." This line needs to be reworded.

**Review by birchy82 — COMMENTED** [review]
Looks good, I left one comment to adjust the opening paragraph in the readme.

File: README.md (hunk 1 above)
Comment (heatherwoz) [reply README.md]: thanks @birchy82 , fixed that and a couple other little things I noticed. ready for re-review.

**Review by birchy82 — APPROVED** [review]
Looks good to me!

**Comment by ccjjmartin** [comment]
@heatherwoz This was already reviewed and merged but I just had a chance to look at it, just wanted to say great job with the docs 👍 and thanks.

**Pull request merged**